src/content/docs/en/guides/routing.mdx

```diff
@@ -52,11 +52,11 @@ Because all routes must be determined at build time, a dynamic route must export
 
 ```astro title="src/pages/dogs/[dog].astro"
 ---
-export function getStaticPaths() {
+export async function getStaticPaths() {
```

Comment:
Any reason to add `async ` to the function even though there's no async code in these samples? I know it's probably minimal, but I suspect suggesting people to add `async` everytime costs on performance.